include/circt/Dialect/SV/SVExpressions.td

```diff
@@ -201,6 +201,18 @@ def ConstantStrOp : SVOp<"constantStr", [Pure]> {
   let assemblyFormat = " $str attr-dict";
 }
 
+def SFormatFOp : SVOp<"sformatf", [Pure]> {
+  let summary = "sformatf";
+  let description = [{
+    This operation produces a constant string literal.
```

Comment:
Saying "constant" may not be correct.

Can you mention `Section 21.3.3 of 1800-2023`. 😉